src/lua/luaTask.c

```diff
@@ -37,14 +37,20 @@
 #include "lua.h"
 #include "lauxlib.h"
 #include "lualib.h"
+#include "GPIO.h"
 
 #include <stdbool.h>
 
+#define ERROR_LED 3
 #define DEFAULT_ONTICK_HZ 1
 #define MAX_ONTICK_HZ 30
 #define LUA_STACK_SIZE 1000
 #define LUA_PERIODIC_FUNCTION "onTick"
 
+#define LUA_BYPASS_FLASH_DELAY 250
+#define LUA_BYPASS_DELAY_SEC 5
+#define LUA_BYPASS_FLASH_COUNT (1000 / LUA_BYPASS_FLASH_DELAY) * LUA_BYPASS_DELAY_SEC
```

Comment:
Also... when using defines everything should be in parens to avoid issues.  So this should read

```C
#define LUA_BYPASS_FLASH_COUNT ((1000 / LUA_BYPASS_FLASH_DELAY) * LUA_BYPASS_DELAY_SEC)
```
or
#define LUA_BYPASS_FLASH_COUNT (LUA_BYPASS_DELAY_SEC * 1000 / LUA_BYPASS_FLASH_DELAY)